nearpy/storage/storage_memory.py

```diff
@@ -49,6 +49,13 @@ def store_vector(self, hash_name, bucket_key, v, data):
             self.buckets[hash_name][bucket_key] = []
         self.buckets[hash_name][bucket_key].append((v, data))
 
+    def delete_vector(self, hash_name, data):
+        """
+        Deletes vector and JSON-serializable data in bucket with specified key.
+        """
+        for bucket_key in self.buckets[hash_name]:
+            self.buckets[hash_name][bucket_key] = [(v, id) for v, id in self.buckets[hash_name][bucket_key] if id != data]
```

Comment:
Anyway, it is possible to add a (v, id) pair as input to identify a set and delete just the one matching.